controllers/rollingupgrade_controller.go

```diff
@@ -151,35 +151,38 @@ func (r *RollingUpgradeReconciler) preDrainHelper(ruObj *upgrademgrv1alpha1.Roll
 // Operates on any scripts that were provided after the draining of the node.
 // kubeCtlCall is provided as an argument to decouple the method from the actual kubectl call
 func (r *RollingUpgradeReconciler) postDrainHelper(ruObj *upgrademgrv1alpha1.RollingUpgrade, nodeName string, kubeCtlCall string) error {
+	logr := r.logger(ruObj)
 	if ruObj.Spec.PostDrain.Script != "" {
-		_, err := runScript(ruObj.Spec.PostDrain.Script, false, ruObj.Name)
+		_, err := r.runScript(ruObj.Spec.PostDrain.Script, false, ruObj)
 		if err != nil {
-			msg := "Failed to run postDrain script: " + err.Error()
-			log.Printf("%s: %s", ruObj.Name, msg)
+			msg := "Failed to run postDrain script: "
+			logr.Error(err, msg)
```

Comment:
If we are sending this error upstream i.e. `return result`, do we also need to log it here? won't it log twice?

Reply:
Most probably, I just replaced logging, didn't cleanup double logging.

Reply:
this error get's propagated to `UpdateInstances` method and then used to build `UpdateInstancesError` which then get logged here as a summary error: https://github.com/keikoproj/upgrade-manager/blob/634036207388b0b5b24e057401a1ca32670bd1fa/controllers/rollingupgrade_controller.go#L670-L674